test/integration/nonce_test.go

```diff
@@ -41,8 +42,13 @@ func TestNonceBalancer_NoBackendMatchingPrefix(t *testing.T) {
 	tlsConfig, err := c.NotWFE.TLS.Load(metrics.NoopRegisterer)
 	test.AssertNotError(t, err, "Could not load TLS config")
 
-	rncKey, err := c.NotWFE.NoncePrefixKey.Pass()
-	test.AssertNotError(t, err, "Failed to load noncePrefixKey")
+	var rncKey []byte
+	rncKey, err = c.NotWFE.NonceHMACKey.Load()
+	if err != nil {
+		rncKeyString, err := c.NotWFE.NoncePrefixKey.Pass()
+		test.AssertNotError(t, err, "Failed to load nonceHMACKey or noncePrefixKey")
+		rncKey = []byte(rncKeyString)
+	}
```

Comment:
We don't need the error condition and fallback here. This config is loaded from test/integration/testdata/nonce-client.json, which, as of this PR, contains `nonceHMACKey`. So we can rely on it existing. We can also remove the `NoncePrefixKey` config field.